src/index.js

```diff
@@ -7,43 +7,61 @@ import transformCall from './transformers/call';
 import transformImport from './transformers/import';
 
 
-const defaultBabelExtensions = ['.js', '.jsx', '.es', '.es6'];
-export const defaultExtensions = defaultBabelExtensions;
+const defaultExtensions = ['.js', '.jsx', '.es', '.es6'];
 
 function isRegExp(string) {
   return string.startsWith('^') || string.endsWith('$');
 }
 
-// The working directory of Atom is the project,
-// while Sublime sets the working project to the directory in which the current opened file is
-// So we need to offer a way to customize the cwd for the eslint plugin
-export function manipulatePluginOptions(pluginOpts, cwd = process.cwd()) {
```

Comment:
Nop that's fine. The previous name was used because of the `manipulatePluginOptions` from Babel ;) Now that's we're not using it anymore, `normalizeOptions` is good enough.
We'll see later if we need to export them. Otherwise, let's keep it as it is now so we won't export functions if they are not needed.